### DISCUSSION TITLE
Add a way to purge the list of recently used nodes inside the Create New Node window.

### DISCUSSION THREAD

**Discussion by drakonbane** [godotengine/godot-proposals#7146] [Editor]
Ive been getting console errors about a node that an addon added that i no longer have installed due to deleted the addon. I suspect its because the editor wants to put it into the recently used nodes list but cant find it. Having a way to clear the recently used nodes list would be convenient anyway. Screenshots included for clarity.

<img width="844" alt="Capture" src="https://github.com/godotengine/godot-proposals/assets/29145604/611d5bb9-84e0-4757-bba0-86b513f91b5b">

**Comment by Calinou** [comment]
- Related to https://github.com/godotengine/godot/issues/35146.
